nvm.sh

```diff
@@ -1545,6 +1574,20 @@ nvm() {
       if [ "_$1" = "_-s" ]; then
         nobinary=1
         shift
+        if [ "_$1" = "_-j" ]; then
+          shift
+          local NVM_CPU_THREAD_VALID
+          NVM_CPU_THREAD_VALID=$(nvm_is_natural_num $1)
+          if [ "$NVM_CPU_THREAD_VALID" = "true" ]; then
+            NVM_MAKE_JOBS=$1
```

Comment:
can we output something to stdout here indicating what the number of threads will be (for the user to confirm they entered it right)?